docs/rst/demos/queens_embedding.rst

```diff
@@ -0,0 +1,261 @@
+    # {Ex0, Ex1, ... Ex(y-1), Ex(y+1), ...} < {Qxy, B}
+    # {E0y, E1y, ... E(x-1)y, E(x+1)y, ...} < {Qxy, B}
+    theEmbedding.REGION = 6
+    for x in range(0, boardDim):
+        for y in range(0, boardDim):
+            Qxy = F("Q", ix(x, y))
+            Ex = F("E(x=, )", x)
+            Ey = F("E( ,y=)", y)
+            ADD(INC(S(R(Ex, F(Ex, y))), M(Qxy, "B")))
+            ADD(INC(S(R(Ey, F(Ey, x))), M(Qxy, "B")))
+
+    # Q produce empty diagonals
+    # E.g. {E(0,1), E(1,2), E(3,4), ...} < Q(2,3)
+    #      {E(0,5), E(1,4), E(3,2), ...} < Q(2,3)
+    theEmbedding.REGION = 7
+
+    for x in range(0, boardDim):
+        for y in range(0, boardDim):
+            diags = []
+            for xx in range(0, boardDim):
+                for yy in range(0, boardDim):
+                    if (x != xx) and (y != yy):
+                        if (x - xx) == (y - yy):
+                            Exxyyy = F("E", ix(xx, yy))
+                            diags.append(Exxyyy)
+            Qxy = F("Q", ix(x, y))
+            if diags:
+                ADD(INC(M(*diags), M(Qxy, "B")))
+
+    for x in range(0, boardDim):
+        for y in range(0, boardDim):
+            diags = []
+            for xx in range(0, boardDim):
+                for yy in range(0, boardDim):
+                    if (x != xx) and (y != yy):
+                        if (x - xx) == -(y - yy):
+                            Exxyyy = F("E", ix(xx, yy))
+                            diags.append(Exxyyy)
+            Qxy = F("Q", ix(x, y))
+            if diags:
+                ADD(INC(M(*diags), M(Qxy, "B")))
+
+    # Set some Queens as boundary conditions
+    theEmbedding.REGION = 9
+    if boardDim == 5:
+        # Set a Queen at the center cell
+        ADD(INC(F("Q", ix(2, 2)), "B"))
+
+    if boardDim == 8:
+        # Set two Queens at correct positions
+        ADD(INC(M(F("Q", ix(1, 4)), F("Q", ix(3, 3))), "B"))
+
+    # Both, empty and queen are not allowed in the solution board
+    # Q(x,y) !< {E(x,y), B} for all (x,y)
+    theEmbedding.REGION = 11
+    ADD(EXC(M(T("Q", _i), T("E", _i)), "B"))
+
+    # Require all columns and all rows with a queen
+    # {R1, R2, ..., C1, C2...} < B
+    theEmbedding.REGION = 12
+    ADD(INC("R", "B"))
+    ADD(INC("C", "B"))
+
+
+
+The T command can be complex to use and it can generally be expanded into a normal loop:
+
+.. code-block:: python
+
+    EXC(T("R", _i), M(R("Q(x=, )", T("Q(x=, )", _i)), "E"))
+
+    term_left = T("R", _i)
+    term_right = M(R("Q(x=, )", T("Q(x=, )", _i)), "E")
+
+The term on the left sets a loop over elements in the R term (representing active Rows)
+
+The term on the right combines the result of an R command with the term E (all Empty cells).
+The R command removes from the Q(x=,) vector, the term at a certain position pointed by the T operator.
+Since both T's use the same index, they move alongside (as in a one-dimensional loop).
```

Comment:
```suggestion
The term on the right combines the result of an ``R`` command with the term ``E`` (all Empty cells).
The ``R`` command removes from the ``Q(x=,)`` vector, the term at a certain position pointed by the ``T`` operator.
Since both ``T``'s use the same index, they move alongside (as in a one-dimensional loop).
```